packages/frontend/src/components/statusboard/Status.vue

```diff
@@ -69,14 +78,42 @@ export default {
       discordLastReady: 'Letzte erfolgreiche Discord Verbindung',
       discordCurrentChannel: 'Aktueller Discord Channel',
     },
+    fetchInProgress: false,
   }),
   methods: {
     onSubmit() {
       // Update data
       this.$store
         .dispatch('getStatus')
         .then(() => {
-          notifySuccess('Aktualisierung erfolgreich!');
+          notifySuccess('Statusdaten aktualisiert!');
+        })
+        .catch((apiResponse) => {
+          if (apiResponse.code) {
+            notifyFailure(apiResponse.error[0].message);
+
+            if (apiResponse.code === 401) {
+              notifyFailure(
+                'Zugang leider abgelaufen! Bitte melde dich erneut an!',
+              );
+              this.$store.dispatch('logout');
+              this.$router.push({ name: 'Login' });
+            }
+          } else {
+            // request failed locally - maybe no internet connection etc?
+            notifyFailure(
+              'Anfrage fehlgeschlagen! Bitte überprüfe deine Internetverbindung.',
+            );
+          }
+        });
+    },
+
+    fetchAndNotify() {
+      this.$store
+        .dispatch('triggerFetch')
+        .then(() => {
+          notifySuccess('Alle Updates wurden erfolgreich von Moodle abgerufen!');
```

Comment:
```suggestion
          notifySuccess('Moodle Updates erfolgreich abgerufen!');
```

Nur ein Vorschlag, finde ich passt etwas besser zum bisherigen Sprachklang :-)